modules/acquia_cms_common/src/AcquiaCmsCommonServiceProvider.php

```diff
@@ -39,7 +39,7 @@ public function register(ContainerBuilder $container) {
    * {@inheritdoc}
    */
   public function alter(ContainerBuilder $container) {
-    if ($container->hasDefinition('acquia_connector.telemetry')) {
+    if ($container->hasDefinition('acquia_connector.telemetry') && PHP_SAPI !== 'cli') {
```

Comment:
&& PHP_SAPI !== 'cli' should be in onTerminateResponse method as here we are altering the service

Reply:
done.